### ISSUE TITLE
Staging Review finding: Accessible name must include visible text

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#129392]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ursa Minor
**Product:** Travel Pay Claims
**Feature:** Complex Claims v1

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.]()
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On the Past Appointments page, the VA Appointment links accessible names don't include the text "VA appointment'. When there is visible text for a control, the accessible name must include that visual text, so that users who use voice control or similar software can effectively activate that control. 

<details><summary>Link, screenshot or steps to recreate</summary>

```
<a href="past/658e0d2d99ce0e4f8d5a0617775821d0c37ed81ff10b7beb4bf1a571d5ce1be3" aria-label="Details for in-person appointment on Monday, December 22 8:00 a.m., Mountain time" class="vaos-appts__focus--hide-outline vaos-hide-for-print">VA appointment</a>
```

<img width="1522" height="403" alt="link text" src="https://github.com/user-attachments/assets/e79c6a46-08b4-4647-8547-838f75360e0f" />

</details>

## Recommended action

Update the `aria-label` to "VA appointment December 22 8:00am"

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG22/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by ala-yna** [comment]
We can take on this one. Seems like we'll want to align with UX first.
